### PR TITLE
chore: remove opa and envoy(MAPCO-3324)

### PR DESCRIPTION
**Pull request by almog8k**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: helm/config/policy.rego [hunk 1]
```diff
@@ -1,6 +1,4 @@
-package envoy.authz
```
Comment (asafMasa) [inline helm/config/policy.rego:1]: remove this file

File: helm/templates/service.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {{- $releaseName := .Release.Name -}}
 {{- $chartName := include "exporter-trigger.name" . -}}
 {{- $cloudProviderFlavor := include "exporter-trigger.cloudProviderFlavor" . -}}
-{{ $targetPort := ternary .Values.envoy.port .Values.env.targetPort .Values.rasterCommon.authentication.opa.enabled }}
+{{ $targetPort := ternary .Values.env.port .Values.env.targetPort .Values.rasterCommon.authentication.opa.enabled }}
```
Comment (asafMasa) [inline helm/templates/service.yaml]: .authentication.opa.enabled ???

File: helm/values.yaml [hunk 3]
```diff
@@ -46,35 +46,6 @@ image:
...
-  requestedDomain: rasterExport
-  resources:
-    enabled: true
-    value:
-      limits:
-        cpu: 100m
-        memory: 128Mi
-      requests:
-        cpu: 100m
-        memory: 128Mi
```
Comment (asafMasa) [inline helm/values.yaml:77]: what is the usage of "internalPort"?

Comment (asafMasa) [reply helm/values.yaml:77]: why do you need "targetPort" if you have "port"?

File: helm/values.yaml [hunk 4]
```diff
@@ -46,35 +46,6 @@ image:
   repository: exporter-trigger
   tag: 'latest'
 
-opa:
```
Comment (asafMasa) [inline helm/values.yaml:49]: do you need "jwtPublicKeySecretName: map-colonies-jwks" ?

File: helm/values.yaml (hunk 3 above)
Comment (asafMasa) [reply helm/values.yaml:77]: rout path: should be /

Comment (asafMasa) [reply helm/values.yaml:77]: remove route definitions

**Review by asafMasa — CHANGES_REQUESTED** [review]
Please mind my requsts

File: helm/templates/service.yaml (hunk 2 above)
Comment (almog8k) [reply helm/templates/service.yaml]: removed

File: helm/values.yaml (hunk 3 above)
Comment (almog8k) [reply helm/values.yaml:77]: done

File: helm/values.yaml (hunk 4 above)
Comment (almog8k) [reply helm/values.yaml:49]: done

**Review by asafMasa — APPROVED** [review]
please test it before merging, good job 👍

**Pull request merged**